--- Data-UUID-0.11/UUID.h.orig	2003-08-27 21:38:35.000000000 +0200
+++ Data-UUID-0.11/UUID.h	2004-05-08 21:00:28.956591060 +0200
@@ -2,6 +2,7 @@
 #    define  __UUID_H__
 
 #include <string.h>
+#include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
@@ -59,11 +60,11 @@
 
 #define CHECK(f1, f2) if (f1 != f2) RETVAL = f1 < f2 ? -1 : 1;
 
-typedef unsigned long      unsigned32;
-typedef unsigned short     unsigned16;
+typedef uint32_t           unsigned32;
+typedef uint16_t           unsigned16;
 typedef unsigned char      unsigned8;
 typedef unsigned char      byte;
-typedef unsigned long long unsigned64_t;
+typedef uint64_t           unsigned64_t;
 typedef unsigned64_t       uuid_time_t;
 
 #if   defined __solaris__ || defined __linux__
